examples/adding_accessories_dynamically.rs

```diff
@@ -1,5 +1,3 @@
-use tokio;
```

Comment:
I think all of these obsolete `tokio` imports are something that should be fixed on the template inside the codegen crate rather than by clippy after the codegen step. This applies to all auto-generated files.